webextensions/api/privacy.json

```
@@ -62,7 +62,7 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": true
+                  "version_added": "≤90"
```

```suggestion
                  "version_added": false
```

The `chrome.privacy.network.httpsOnlyMode` API does not exist in Chrome. It is not supported in Chromium browsers either.